README.md

```diff
@@ -1,81 +1,127 @@
+```hcl
+terraform {
+  required_providers { iterative = { source = "iterative/iterative" } }
+}
+provider "iterative" {}
+resource "iterative_task" "example" {
+  cloud      = "aws" # or any of: gcp, az, k8s
+  machine    = "m"   # medium. Or any of: l, xl, m+k80, xl+v100, ...
+  spot       = 0     # auto-price. Or -1 to disable, or >0 to set a hourly USD limit
+  disk_size  = 30    # GB
+
+  storage {
+    workdir = "."
+    output  = "results"
+  }
+  script = <<-END
+    #!/bin/bash
+    mkdir results
+    echo "Hello World!" > results/greeting.txt
+  END
+}
+```
 
-## Development
+See [the reference](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#argument-reference) for the full list of options for `main.tf` -- including more information on [`machine` types](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#machine-type) with and without GPUs.
 
-### Install Go 1.17+
+Run this once (in the directory containing `main.tf`) to download the `required_providers`:
 
-Refer to the [official documentation](https://golang.org/doc/install) for specific instructions.
+```
+terraform init
+```
 
-### Clone the repository
+### Run Task
 
-```console
-git clone https://github.com/iterative/terraform-provider-iterative
-cd terraform-provider-iterative
+```
+terraform apply
 ```
 
-### Install the provider
+This launches a `machine` in the `cloud`, uploads `workdir`, and runs the `script`. Upon completion (or error), the `machine` is terminated.
 
-Build the provider and install the resulting binary to the [local mirror directory](https://www.terraform.io/docs/cli/config/config-file.html#implied-local-mirror-directories):
+With spot/preemptible instances (`spot >= 0`), auto-recovery logic and persistent storage will be used to relaunch interrupted tasks.
 
-```console
-make install
-```
+### Query Status
+
+Results and logs are periodically synced to persistent cloud storage. To query this status and view logs:
 
-### Create a test file
+```
+terraform refresh
+terraform show
+```
 
-Create a file named `main.tf` in an empty directory with the following contents:
+### Stop Tasks
 
-```hcl
-terraform {
-  required_providers { iterative = { source = "iterative/iterative" } }
-}
-provider "iterative" {}
-# ... other resource blocks ...
+```
+terraform destroy
 ```
 
-**Note:** to use your local build, specify `source = "github.com/iterative/iterative"` (`source = "iterative/iterative"` will download the latest stable release instead).
+This terminates the `machine` (if still running), downloads `output`, and removes the persistent `disk_size` storage.
 
-### Initialize the provider
+## Help
 
-Run this command after every `make install` to use the new build:
+The [getting started guide](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) has some more information.
```

Comment:
> [The GS] has some more information.

Does it? They look vey similar at first glance. Why refer to almost the same doc but in another site? Hard to find meaningful differences

Reply:
Could instead link from relevant places in the README to the other docs, for now Azure Kubernetes Service and Generic Machine Types.